Source/Scene/GlobeSurfaceTileProvider.js

```diff
@@ -395,7 +399,7 @@ define([
 
         var cullingVolume = frameState.cullingVolume;
 
-        var boundingVolume = surfaceTile.boundingSphere3D;
+        var boundingVolume = defaultValue(surfaceTile.boundingOBB, surfaceTile.boundingSphere3D);
 
         if (frameState.mode !== SceneMode.SCENE3D) {
```

Comment:
I haven't done anything to make it use OBBs in 2D and Columbus at all - it still falls back to using bounding spheres in exactly the same way as before. I guess that's something to add in.